src/sidebar/services/test/groups-test.js

```diff
@@ -106,12 +114,12 @@ describe('groups', function() {
         },
       },
       groups: {
-        list: sandbox.stub().returns(
-          Promise.resolve({
-            data: dummyGroups,
-            token: '1234',
-          })
-        ),
+        list: sandbox.stub().returns(Promise.resolve(dummyGroups)),
```

Comment:
This is fine. I noticed looking in the docs that current versions of Sinon have a `resolves(value)` method which can be used as a shorthand for `returns(Promise.resolve(value))`.